org.btchd.btchd-qt.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.btchd.btchd-qt</id>
+  <name>BitcoinHD Core</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>
+    Connect to the BitcoinHD P2P network
+  </summary>
+  <url type="homepage">
+    https://btchd.org/
+  </url>
+  <url type="bugtracker">
+    https://github.com/bitcoinhd/bitcoinhd/issues/new?title=[flatpak]
+  </url>
+  <launchable type="desktop-id">
+    org.btchd.btchd-qt.desktop
+  </launchable>
+  <description>
+    <p>
+    Bitcoin is an experimental digital currency that enables instant
+    payments to anyone, anywhere in the world. Bitcoin uses
+    peer-to-peer technology to operate with no central authority:
+    managing transactions and issuing money are carried out
+    collectively by the network. Bitcoin Core is the name of open
+    source software which enables the use of this currency.
+    </p>
```

Comment:
I would add a section here explaining how BTCHD is different from regular Bitcoin.

Reply:
Done.